osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/management-ws-smartmetering.xsd

```diff
@@ -273,6 +273,7 @@
   <xsd:element name="ClearMbusStatusRequest">
     <xsd:complexType>
       <xsd:sequence>
+        <xsd:element name="DeviceIdentification" type="common:DeviceIdentification"/>
         <xsd:element name="ClearMbusStatusRequestData"
           type="tns:ClearMbusStatusRequestData"/>
       </xsd:sequence>
```

Comment:
This is actually a comment on the ClearMbusStatusResponse: Why is ClearMbusStatusResponse containing a ClearMbusStatusResponseData object while this ResponseData object is not more than a extension of the basic Response class
 
```
   <xsd:element name="ClearMbusStatusResponse">
    <xsd:complexType>
      <xsd:sequence>
        <xsd:element name="Result" type="common:OsgpResultType"/>
        <xsd:element name="ClearMbusStatusResponseData"
          type="tns:ClearMbusStatusResponseData"/>
      </xsd:sequence>
    </xsd:complexType>
  </xsd:element>
  
  <xsd:complexType name="ClearMbusStatusResponseData">
    <xsd:complexContent>
      <xsd:extension base="common:Response">
      </xsd:extension>
    </xsd:complexContent>
  </xsd:complexType>
```

Could it not be simplyfied to something like this?

```
  <xsd:element name="ClearMbusStatusResponse">
    <xsd:complexType>
      <xsd:sequence>
        <xsd:element name="Result" type="common:OsgpResultType"/>
        <xsd:element name="Description" type="xsd:string" minOccurs="0"/>
      </xsd:sequence>
    </xsd:complexType>
  </xsd:element>
```

Reply:
This construct is used for Bundled responses, but for responses without data we actually do use the construct you suggest. I will change this so it is consistent and can profit of the generic display logic in the SMHE GUI.